mlflow/store/tracking/rest_store.py

```diff
@@ -291,6 +311,9 @@ def record_logged_model(self, run_id, mlflow_model):
         self._call_endpoint(LogModel, req_body)
 
 
+_LIST_EXPERIMENTS_MAX_RESULTS_DEFAULT_IN_DATABRICKS = 1000
```

Comment:
Unsure about the convention to put DB specific constant. Perhaps move it to the top of the file for better readability?

Also what's the reason to choose a separate parameter instead of using SEARCH_MAX_RESULTS_DEFAULT?

Reply:
> Also what's the reason to choose a separate parameter instead of using SEACH_MAX_RESULTS_DEFAULT?

To avoid the case where someones updates `SEARCH_MAX_RESULTS_DEFAULT` without knowing it's also used in `DatabricksRestStore`